mlnlffigen/ast-to-spec.sml

```diff
@@ -377,7 +386,7 @@ structure AstToSpec = struct
               args = case args of
                          [(arg, _)] => (case getCoreType arg of
                                        A.Void => []
-                                     | _ => [valty_nonvoid C arg])
+                                     | _ => [fun_valty_nonvoid C arg])
                        | _ => let fun build [] = []
                                     | build [(x, _)] =
                                       ([valty_nonvoid C x]
```

Comment:
Should use `fun_valty_nonvoid` here to check if the last argument is a struct/union.